sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/SignatureAlgorithm.cs

```diff
@@ -35,55 +36,60 @@ public SignatureAlgorithm(string value)
         }
 
         /// <summary>
-        /// Gets an RSA SHA-256 <see cref="SignatureAlgorithm"/>.
+        /// Gets an RSA SHA-256 <see cref="SignatureAlgorithm"/> as described in <see href="https://tools.ietf.org/html/rfc7518"/>.
```

Comment:
should we make this an aka.ms link?

Reply:
I don't think that's necessary. These IETF links have been this way for as long as I can remember, and I'm old.